pkg/backup/backupdest/backup_destination.go

```diff
@@ -237,41 +237,40 @@ func ResolveDest(
 	}
 	prevBackupURIs = append([]string{plannedBackupDefaultURI}, prevBackupURIs...)
 
-	// Within the chosenSuffix dir, differentiate incremental backups with partName.
-	partName := endTime.GoTime().Format(backupbase.DateBasedIncFolderName)
-	if execCfg.Settings.Version.IsActive(ctx, clusterversion.V25_2) {
```

Comment:
nit: looks like you're removing a version gate in this part of the commit. I think this should be a separate commit.